### PR TITLE
Replace Logos with cleaned-up versions

### PR DESCRIPTION
**Pull request by louisescher**
#### Description

- This PR moves the old logos to `/www/assets/old` and replaces them with newer, cleaner versions. This allows for smaller SVG files and better rounded corners, since the old ones had some issues with the rounding. Also, new outlined versions have been added as per @Adammatthiesen's request.

### CODE REVIEW

File: www/assets/logo-discord.png
Comment (Adammatthiesen) [inline www/assets/logo-discord.png:1]: Could you make the actual logo on this a bit smaller, the image has to fit within the discord profile image rounded border and would get cutoff

File: www/assets/logo-discord.svg
Comment (Adammatthiesen) [inline www/assets/logo-discord.svg:1]: (same as the .png version)
Could you make the actual logo on this a bit smaller, the image has to fit within the discord profile image rounded border and would get cutoff

**Review by Adammatthiesen — CHANGES_REQUESTED** [review]

**Review by Adammatthiesen — APPROVED** [review]

**Review by dreyfus92 — APPROVED** [review]
you're moving at the speed of light @louisescher 🫡

**Comment by louisescher** [comment]
LOOOTS of free time 😂

**Pull request merged**